http-clients/src/test/java/org/triplea/http/client/web/socket/WebSocketConnectionTest.java

```diff
@@ -61,59 +55,36 @@ void onError() {
   @Nested
   class SendMessageAndConnect {
     @Mock private WebSocketClient webSocketClient;
-    @Mock private WebSocket webSocket;
-    @Mock private WebSocketConnector webSocketConnector;
 
     private WebSocketConnection webSocketConnection;
 
     @BeforeEach
     void setup() {
       webSocketConnection = new WebSocketConnection(LOCALHOST_URI);
       webSocketConnection.setClient(webSocketClient);
-      webSocketConnection.setWebSocketConnector(webSocketConnector);
     }
 
     @Test
-    void connect() {
+    void connect() throws Exception {
       webSocketConnection.connect();
 
-      verify(webSocketConnector).initiateConnection();
+      verify(webSocketClient)
+          .connectBlocking(
+              WebSocketConnection.DEFAULT_CONNECT_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
```

Comment:
Very nitpicky, I'm not sure if we gain much by verifying the parameters of this test, we could make this more flexible by just asserting that any arg is passed to connect blocking.